CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ The next release will require at least [Go 1.24].
 ### Changed
 
 - Change the default span name to be `GET /path` so it complies with the HTTP semantic conventions in `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`. (#7551)
+- Change the faas.max_memory unit to be bytes instead of MB to comply with the semantic conventions in `"go.opentelemetry.io/contrib/detectors/aws/lambda`. (#7745)
```

Comment:
```suggestion
- Change the `faas.max_memory` unit to be bytes instead of MB to comply with the semantic conventions in `go.opentelemetry.io/contrib/detectors/aws/lambda`. (#7745)
```